### ISSUE TITLE
Restoring backup of multi-node cluster to single-node instance fails on restart

### ISSUE THREAD

**Issue by michaelyork** [flynn/flynn#3669]
Have tested this a few times now over the past 24 hours. Running a single Ubuntu 14.04 instance via Hyper-V. When the VM is restarted, Flynn gets into an infinite loop seemingly trying to start the controller.

```
t=2016-11-13T12:54:36-0800 lvl=eror msg="error repairing cluster" component=cluster-monitor fn=checkCluster err="discoverd: timed out waiting for instances"
t=2016-11-13T12:54:36-0800 lvl=eror msg="did not find any controller api instances" component=cluster-monitor fn=checkCluster
t=2016-11-13T12:54:36-0800 lvl=eror msg="scheduler is not up" component=cluster-monitor fn=checkCluster
t=2016-11-13T12:54:36-0800 lvl=eror msg="fault deadline reached" component=cluster-monitor fn=checkCluster
t=2016-11-13T12:54:36-0800 lvl=info msg="initiating cluster repair" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:54:36-0800 lvl=info msg="killing any running schedulers to prevent interference" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:54:36-0800 lvl=info msg="request started" component=host req_id=54798a1e-c217-44d4-a76a-5e84130d1edb method=GET path=/host/jobs client_ip=192.168.128.87
t=2016-11-13T12:54:36-0800 lvl=info msg="request completed" component=host req_id=54798a1e-c217-44d4-a76a-5e84130d1edb status=200 duration=2.5827ms
t=2016-11-13T12:54:36-0800 lvl=info msg="checking status of sirenia databases" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:54:36-0800 lvl=info msg="checking for database state" component=cluster-monitor fn=repairCluster db=postgres
t=2016-11-13T12:54:36-0800 lvl=info msg="checking sirenia cluster status" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:54:36-0800 lvl=info msg="found running leader" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:54:36-0800 lvl=info msg="found running instances" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres count=1
t=2016-11-13T12:54:36-0800 lvl=info msg="getting sirenia status" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:54:36-0800 lvl=info msg="getting service metadata" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:54:36-0800 lvl=info msg="getting primary job info" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres job.id=apps-e4f514e3-8b44-48bc-982d-84ea455d71fc
t=2016-11-13T12:54:36-0800 lvl=info msg="request started" component=host req_id=96ef3c3b-1dda-4a0f-99b4-4d002c79660d method=GET path=/host/jobs/apps-e4f514e3-8b44-48bc-982d-84ea455d71fc client_ip=192.168.128.87
t=2016-11-13T12:54:36-0800 lvl=info msg="request completed" component=host req_id=96ef3c3b-1dda-4a0f-99b4-4d002c79660d status=200 duration=98.9µs
t=2016-11-13T12:54:36-0800 lvl=info msg="getting sync job info" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres job.id=apps-8fc867a7-dd2e-48fd-902c-e19076ffaf8c
t=2016-11-13T12:54:36-0800 lvl=info msg="request started" component=host req_id=0a1c819d-fe49-43e7-8797-cdc7f47c82e7 method=GET path=/host/jobs/apps-8fc867a7-dd2e-48fd-902c-e19076ffaf8c client_ip=192.168.128.87
t=2016-11-13T12:54:36-0800 lvl=info msg="request completed" component=host req_id=0a1c819d-fe49-43e7-8797-cdc7f47c82e7 status=200 duration=90.7µs
t=2016-11-13T12:54:36-0800 lvl=info msg="terminating unassigned sirenia instances" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:54:36-0800 lvl=info msg="checking for database state" component=cluster-monitor fn=repairCluster db=mariadb
t=2016-11-13T12:54:36-0800 lvl=info msg="skipping recovery of db, no state in discoverd" component=cluster-monitor fn=repairCluster db=mariadb
t=2016-11-13T12:54:36-0800 lvl=info msg="no controller web process running, getting release details from hosts" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:54:36-0800 lvl=info msg="request started" component=host req_id=e24363aa-afa7-4311-b178-d7749467389c method=GET path=/host/jobs client_ip=192.168.128.87
t=2016-11-13T12:54:36-0800 lvl=info msg="request completed" component=host req_id=e24363aa-afa7-4311-b178-d7749467389c status=200 duration=3.3356ms
t=2016-11-13T12:54:36-0800 lvl=info msg="starting controller web job" component=cluster-monitor fn=repairCluster job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a release=81da55ea-caeb-438f-a8f0-2b60ce0c2e57
t=2016-11-13T12:54:36-0800 lvl=info msg="request started" component=host req_id=e5341a29-fb84-470f-bb3a-c0a89bb8f613 method=PUT path=/host/jobs/apps-4e48dbd2-6c75-4472-ac14-541ccab1451a client_ip=192.168.128.87
t=2016-11-13T12:54:36-0800 lvl=info msg="decoding job" app=host pid=1305 host.id=apps fn=AddJob job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="acquiring state database" app=host pid=1305 host.id=apps fn=AddJob job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="request completed" component=host req_id=e5341a29-fb84-470f-bb3a-c0a89bb8f613 status=200 duration=4.3909ms
t=2016-11-13T12:54:36-0800 lvl=info msg="running job" app=host pid=1305 host.id=apps fn=AddJob job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="starting job" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a job.artifact.uri="https://dl.flynn.io/tuf?name=flynn/controller&id=05c80cdbc06f4e0025d063d61a37cbf18a8617624fbe51f04fd59fe9f5b3df6a" job.args="[/bin/start-flynn-controller controller]"
t=2016-11-13T12:54:36-0800 lvl=info msg="obtained ip" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a network=100.100.50.0/24 ip=100.100.50.16
t=2016-11-13T12:54:36-0800 lvl=info msg="waiting for job to start" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:54:36-0800 lvl=info msg="pulling image" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="reading image config" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="checking out image" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="setting max fd limit to 10000 / 10000" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="mounting container directories and files" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="writing config" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="job started" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="start watching container" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="getting stdout" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="watching for changes" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="state change" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a state=initial
t=2016-11-13T12:54:36-0800 lvl=info msg="waiting for attach" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg=resuming app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg=resumed app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:54:36-0800 lvl=info msg="state change" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a state=running
t=2016-11-13T12:54:36-0800 lvl=info msg="container running" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-4e48dbd2-6c75-4472-ac14-541ccab1451a
t=2016-11-13T12:55:36-0800 lvl=eror msg="error repairing cluster" component=cluster-monitor fn=checkCluster err="discoverd: timed out waiting for instances"
t=2016-11-13T12:55:36-0800 lvl=eror msg="did not find any controller api instances" component=cluster-monitor fn=checkCluster
t=2016-11-13T12:55:36-0800 lvl=eror msg="scheduler is not up" component=cluster-monitor fn=checkCluster
t=2016-11-13T12:55:36-0800 lvl=eror msg="fault deadline reached" component=cluster-monitor fn=checkCluster
t=2016-11-13T12:55:36-0800 lvl=info msg="initiating cluster repair" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:55:36-0800 lvl=info msg="killing any running schedulers to prevent interference" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:55:36-0800 lvl=info msg="request started" component=host req_id=ed12eb01-cd78-4d9b-abb1-4a2e2ae99ed0 method=GET path=/host/jobs client_ip=192.168.128.87
t=2016-11-13T12:55:36-0800 lvl=info msg="request completed" component=host req_id=ed12eb01-cd78-4d9b-abb1-4a2e2ae99ed0 status=200 duration=7.3808ms
t=2016-11-13T12:55:36-0800 lvl=info msg="checking status of sirenia databases" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:55:36-0800 lvl=info msg="checking for database state" component=cluster-monitor fn=repairCluster db=postgres
t=2016-11-13T12:55:36-0800 lvl=info msg="checking sirenia cluster status" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:55:36-0800 lvl=info msg="found running leader" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:55:36-0800 lvl=info msg="found running instances" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres count=1
t=2016-11-13T12:55:36-0800 lvl=info msg="getting sirenia status" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:55:36-0800 lvl=info msg="getting service metadata" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:55:36-0800 lvl=info msg="getting primary job info" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres job.id=apps-e4f514e3-8b44-48bc-982d-84ea455d71fc
t=2016-11-13T12:55:36-0800 lvl=info msg="request started" component=host req_id=b548811d-6573-4f75-903e-ebfdd538d4a5 method=GET path=/host/jobs/apps-e4f514e3-8b44-48bc-982d-84ea455d71fc client_ip=192.168.128.87
t=2016-11-13T12:55:36-0800 lvl=info msg="request completed" component=host req_id=b548811d-6573-4f75-903e-ebfdd538d4a5 status=200 duration=113.5µs
t=2016-11-13T12:55:36-0800 lvl=info msg="getting sync job info" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres job.id=apps-8fc867a7-dd2e-48fd-902c-e19076ffaf8c
t=2016-11-13T12:55:36-0800 lvl=info msg="request started" component=host req_id=3f8b9f8a-e261-4bd7-9b3c-3b0f7a4a1fe0 method=GET path=/host/jobs/apps-8fc867a7-dd2e-48fd-902c-e19076ffaf8c client_ip=192.168.128.87
t=2016-11-13T12:55:36-0800 lvl=info msg="request completed" component=host req_id=3f8b9f8a-e261-4bd7-9b3c-3b0f7a4a1fe0 status=200 duration=89.2µs
t=2016-11-13T12:55:36-0800 lvl=info msg="terminating unassigned sirenia instances" component=cluster-monitor fn=repairCluster fn=FixSirenia service=postgres
t=2016-11-13T12:55:36-0800 lvl=info msg="checking for database state" component=cluster-monitor fn=repairCluster db=mariadb
t=2016-11-13T12:55:36-0800 lvl=info msg="skipping recovery of db, no state in discoverd" component=cluster-monitor fn=repairCluster db=mariadb
t=2016-11-13T12:55:36-0800 lvl=info msg="no controller web process running, getting release details from hosts" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:55:36-0800 lvl=info msg="request started" component=host req_id=c0e78c1a-3a4a-4c4a-8dc9-a17cf99e3c21 method=GET path=/host/jobs client_ip=192.168.128.87
t=2016-11-13T12:55:36-0800 lvl=info msg="request completed" component=host req_id=c0e78c1a-3a4a-4c4a-8dc9-a17cf99e3c21 status=200 duration=2.341ms
t=2016-11-13T12:55:36-0800 lvl=info msg="starting controller web job" component=cluster-monitor fn=repairCluster job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f release=81da55ea-caeb-438f-a8f0-2b60ce0c2e57
t=2016-11-13T12:55:36-0800 lvl=info msg="request started" component=host req_id=423f11c1-0b96-4742-886f-62adf0e20ba6 method=PUT path=/host/jobs/apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f client_ip=192.168.128.87
t=2016-11-13T12:55:36-0800 lvl=info msg="decoding job" app=host pid=1305 host.id=apps fn=AddJob job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="acquiring state database" app=host pid=1305 host.id=apps fn=AddJob job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="request completed" component=host req_id=423f11c1-0b96-4742-886f-62adf0e20ba6 status=200 duration=3.7106ms
t=2016-11-13T12:55:36-0800 lvl=info msg="running job" app=host pid=1305 host.id=apps fn=AddJob job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="starting job" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f job.artifact.uri="https://dl.flynn.io/tuf?name=flynn/controller&id=05c80cdbc06f4e0025d063d61a37cbf18a8617624fbe51f04fd59fe9f5b3df6a" job.args="[/bin/start-flynn-controller controller]"
t=2016-11-13T12:55:36-0800 lvl=info msg="obtained ip" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f network=100.100.50.0/24 ip=100.100.50.17
t=2016-11-13T12:55:36-0800 lvl=info msg="waiting for job to start" component=cluster-monitor fn=repairCluster
t=2016-11-13T12:55:36-0800 lvl=info msg="pulling image" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="reading image config" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="checking out image" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="setting max fd limit to 10000 / 10000" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="mounting container directories and files" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="writing config" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="job started" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=run job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="start watching container" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="getting stdout" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="watching for changes" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="state change" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f state=initial
t=2016-11-13T12:55:36-0800 lvl=info msg="waiting for attach" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg=resuming app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg=resumed app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
t=2016-11-13T12:55:36-0800 lvl=info msg="state change" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f state=running
t=2016-11-13T12:55:36-0800 lvl=info msg="container running" app=host pid=1305 host.id=apps component=backend backend=libcontainer fn=watch job.id=apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f
```

**Comment by michaelyork** [comment]
For reference, here is the output of `flynn-host ps`:

```
michaelyork@apps:~$ flynn-host ps
ID                                         STATE    CREATED             CONTROLLER APP  CONTROLLER TYPE  ERROR
apps-e3448188-7920-4e84-a228-d081bb04b4b6  running  50 seconds ago      controller      web              
apps-1c92627d-a1fb-43c9-a1a1-7c17b1c17d7f  running  About a minute ago  controller      web              
apps-4e48dbd2-6c75-4472-ac14-541ccab1451a  running  2 minutes ago       controller      web              
apps-b66535b9-474c-47f2-9d45-8d3b6c71b0c0  running  3 minutes ago       controller      web              
apps-bc15587d-e191-4d06-adfa-353e5466ce7f  running  4 minutes ago       controller      web              
apps-c8b9e6ac-545c-46a7-87d7-129c82d8f327  running  5 minutes ago       controller      web              
apps-72e9827b-40b6-465d-afc9-4750b53af909  running  6 minutes ago       controller      web              
apps-c0bafacd-7b86-4a6b-8b2b-931b6b96fa0c  running  7 minutes ago       controller      web              
apps-a683faa3-fd02-4d7a-aca6-c1421626f755  running  8 minutes ago       controller      web              
apps-200995fc-a1e5-433e-a214-c8f9e5ee0b0a  running  9 minutes ago       controller      web              
apps-96e1c93d-f79d-43c0-8ebc-afdd04f8c328  running  10 minutes ago      controller      web              
apps-3a908e65-8779-42d8-ab2e-96d1fe06953f  running  11 minutes ago      controller      web              
apps-583bc9a8-76d7-4180-991c-d35e3cfd593f  running  12 minutes ago      controller      web              
apps-635076e1-094c-4071-b830-1a1800cd5ea9  running  13 minutes ago      controller      web              
apps-d29d8628-51d9-4aa1-b1a2-be1e6469a4f5  running  14 minutes ago      controller      web              
apps-e4f514e3-8b44-48bc-982d-84ea455d71fc  running  19 minutes ago      postgres        postgres         
apps-a435fd0b-4254-4f48-8857-bd97b72a5588  running  21 minutes ago      discoverd       app              
apps-4e171b96-0945-4c1b-84de-e205e4602931  running  21 minutes ago      flannel         app              
```

**Comment by michaelyork** [comment]
Seems related to #3640.

**Comment by lmars** [comment]
@michaelyork please run `sudo flynn-host collect-debug-info` and post the resulting gist here (or send to logs@flynn.io).

**Comment by michaelyork** [comment]
https://gist.github.com/5dc62e15e0d208ae43d69ac25db6d2b9

**Comment by lmars** [comment]
The issue here is that the cluster was booted from a backup which came from a multi-node cluster, and so on reboot postgres is transitioning back to multi-node mode as `ENV["SINGLETON"]` is not set.

The easiest fix here is to restore to a multi-node cluster, use `flynn export` to export individual apps, boot a fresh single node cluster and import the apps with `flynn import`.

**Comment by michaelyork** [comment]
Thanks so much for the help today, @lmars. Updated title to track this bug.

**Comment by titanous** [comment]
This was fixed by #3691
